### ISSUE TITLE
No way of turning off afk mode for players

### ISSUE THREAD

**Issue by Dragonissa** [EfeDursun125/TF2-EBOTS#19]
I dont intend to run this plugin for afk players.

- flag for players that should be ignored by the afk check
- cvar to disable the afk mode for players completely

**Comment by EfeDursun125** [comment]
i have added some commands for it, are you using ebots over tfbots?

**Comment by Dragonissa** [comment]
Yes, i hoped i can stick to ebots though i had more problems with them than normal tfbots, mainly them just randomly stop moving when respawned (any class)
